apps/condo/domains/ticket/schema/TicketContact.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Generated by `createschema ticket.TicketContact 'property:Relationship:Property:PROTECT; unitName:Text; email:Text; phone:Text; name:Text;' --force`
+ */
+
+const { Text, Relationship } = require('@keystonejs/fields')
+const { GQLListSchema } = require('@core/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted } = require('@core/keystone/plugins')
+const { SENDER_FIELD, DV_FIELD } = require('@condo/domains/common/schema/fields')
+const access = require('@condo/domains/ticket/access/TicketContact')
+
+
+const TicketContact = new GQLListSchema('TicketContact', {
+    schemaDoc: 'Contact information of a person, who is initiator of ticket(s)',
+    fields: {
+        dv: DV_FIELD,
+        sender: SENDER_FIELD,
+
+        property: {
```

Comment:
could you please also add an organization denormalization field. It's better to use it for access check